File: src/main/java/com/thoo/api/services/AESService.java
package com.thoo.api.services;

import com.thoo.api.model.AESModel;
import com.thoo.api.model.BaseModel;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface AESService {

    String BASE_URL = "https://fortnite-api.com/";

    @GET("/v2/aes")
    Call<BaseModel<AESModel>> aes();

    @GET("/v2/aes")
    Call<BaseModel<AESModel>> aes(@Query("keyFormat") String keyFormat);

}

File: src/main/java/com/thoo/api/services/AESService.kt
package com.thoo.api.services

import com.thoo.api.models.AESModel
import com.thoo.api.models.BaseModel
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Header

interface AESService {

    @GET("/v2/aes")
    fun aes(@Header("keyFormat") format: String = "hex"): Call<BaseModel<AESModel>>

}